src/sider/historikk/Historikk.tsx

```diff
@@ -172,7 +172,14 @@ export function Historikk({ historikkEvents, tilfeller }: Props): ReactElement {
               .sort(byTidspunkt())
               .map(({ expandableContent, kilde, tekst, tidspunkt }, i) => {
                 return expandableContent ? (
-                  <Table.ExpandableRow key={i} content={expandableContent}>
+                  <Table.ExpandableRow
+                    key={i}
+                    content={
+                      <span className={`preserve-whitespace`}>
```

Comment:
Her har vi pleid å bruker tailwind 😊
```suggestion
                      <span className='whitespace-pre-wrap'>
```

Reply:
Skjønner. Var ikke klar over hvordan tailwind fungerte, så fint at den tydeligvis oppretter egne style classes for dette. Da skal det i hvert fall være endret begge steder.